### DISCUSSION TITLE
Example configuration with LDAP for Authentik?

### DISCUSSION THREAD

**Discussion by thcrt** [ArchiveBox/ArchiveBox#1380] [Q&A] [answered]
I want to use Authentik for SSO. ArchiveBox doesn't support OpenID so it has to use proxy auth plus LDAP. I have no experience with LDAP so I was hoping someone could share their working configuration or some information on how to set it up. Cheers :D

**Comment by pirate** [comment]
Try this: https://piotrkrzyzek.com/how-to-setup-use-authentik-with-simple-forward-proxy/

**Reply by thcrt** [reply]
This handles proxy auth, which I already have set up without issue. The problem is that it means I need to authenticate twice with two potentially different sets of credentials. This would be solved by LDAP, which is the part I need help with :)

**Reply by pirate** [reply, marked as answer]
Ah ok, you may want to read/subscribe to this issue https://github.com/ArchiveBox/ArchiveBox/pull/866#issuecomment-1962130200

**Reply by thcrt** [reply]
Alright, thanks :)
